node/src/actors/session/actor.rs

```diff
@@ -31,6 +31,11 @@ impl Actor for Session {
                     act.remote_addr
                 );
```

Comment:
Above this line there is a "Handshake timeout" log in INFO level, it should probably be DEBUG?